juego.py

```diff
@@ -66,87 +70,101 @@ def __init__(self):
-                    elif event.pos[0] > 550 and event.pos[0] < 450 + quit.get_width() and \
-                         event.pos[1] > 300 and event.pos[1] < 300 + quit.get_height():
+                    elif event.pos[0] > sx(550) and event.pos[0] < sx(450) + quit.get_width() and \
+                         event.pos[1] > sy(300) and event.pos[1] < sy(300) + quit.get_height():
                          quit = self.fuente_130.render("SALIR",True,(0,0,255)) 
                          if sonido_menu != None: 
                              sonido_menu.play()
                 elif event.type == MOUSEBUTTONDOWN:
                     if event.button == 1:
-                        if event.pos[0] > 450 and event.pos[0] < 450 + jugar.get_width() and \
-                            event.pos[1] > 100 and event.pos[1] < 100 + jugar.get_height():
+                        if event.pos[0] > sx(450) and event.pos[0] < sx(450) + jugar.get_width() and \
+                            event.pos[1] > sy(100) and event.pos[1] < sy(100) + jugar.get_height():
                             return chosen_level
-                        elif event.pos[0] > 450 and event.pos[0] < 450 + level.get_width() and \
-                            event.pos[1] > 200 and event.pos[1] < 200 + level.get_height():
+                        elif event.pos[0] > sx(450) and event.pos[0] < sx(450) + level.get_width() and \
+                            event.pos[1] > sy(200) and event.pos[1] < sy(200) + level.get_height():
                             chosen_level = self.choose_level()
-                        elif event.pos[0] > 450 and event.pos[0] < 450 + quit.get_width() and \
-                            event.pos[1] > 300 and event.pos[1] < 300 + quit.get_height():
-                            exit()   
+                        elif event.pos[0] > sx(450) and event.pos[0] < sx(450) + quit.get_width() and \
+                            event.pos[1] > sy(300) and event.pos[1] < sy(300) + quit.get_height():
+                            self.running = False
+                            exit()  
             pygame.display.update()
 
 
     def choose_level(self):
         
-        sonido_menu = load_sound("./data/menu.ogg")
+        sonido_menu = load_sound("menu.ogg")
         facil = self.fuente_130.render("facil",True,(0,0,255))
         medio = self.fuente_130.render("medio",True,(0,0,255))
         dificil = self.fuente_130.render("dificil",True,(0,0,255))
         fondo = cargar_imagen('data/1.jpg')
         level = "facil"
-        while 1:
+        while self.running:
             self.screen.fill((0,0,0))
             self.screen.blit(fondo, (0, 0))
-            self.screen.blit(facil,(450,100))
-            self.screen.blit(medio,(450,200))
-            self.screen.blit(dificil,(450,300))
+            self.screen.blit(facil,(sx(450), sy(100)))
+            self.screen.blit(medio,(sx(450), sy(200)))
+            self.screen.blit(dificil,(sx(450), sy(300)))
```

Comment:
Spacing between the first positional argument and the second positional argument might improve readability (`flake`)

Reply:
yes, but let's not ask for that now, as the existing code is not already compliant, so the pull request will become unreasonably complicated.  i'd prefer existing style is kept until a later reformatting is done, separate to all the other work on an activity.

Reply:
I think for now I'm going with the style activity code is written in, and would reformat later like @quozl suggested. Resolving all comma spacing and style related conversations.